msal/src/test/java/com/microsoft/identity/client/CommandParametersTest.java

```diff
@@ -181,94 +180,110 @@ public void testDeviceCodeFlowOperationWithoutClaims() throws ClientException {
 
     @Test
     public void testAppendToExtraQueryParametersIfWebAuthnCapable_UnsetPropertyAndNullInput() {
-        final List<Map.Entry<String, String>> combinedQueryParameters = CommandParametersAdapter.appendToExtraQueryParametersIfWebAuthnCapable(
-                null,
-                getConfiguration(AAD_NONE_CONFIG_FILE)
-        );
-        Assert.assertNull(combinedQueryParameters);
+        if (FidoConstants.IS_PASSKEY_SUPPORT_READY) {
```

Comment:
So these tests will only run if passkey support is ready? 

You may want to use JUnit Assume and assume the value of `IS_PASSKEY_SUPPORT_READY` - this way you can halt (ignore) the test if passkey support is not ready

https://junit.org/junit4/javadoc/4.12/org/junit/Assume.html#assumeTrue(boolean)

Reply:
Thanks for the suggestion; changes here: 773e61e49efec38645c1d19513c3d78e6991c5b8